### PR TITLE
Btfsbtt 663: new metadata implementation

### PR DESCRIPTION
**Pull request by steveyeom**
Added additional method Chunksize() to Splitter interface for the new metadata implementation.

### CODE REVIEW

File: splitting.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ const DefaultBlockSize int64 = 1024 * 256
 type Splitter interface {
 	Reader() io.Reader
 	NextBytes() ([]byte, error)
+	Chunksize() uint32
```
Comment (taiyangc) [inline splitting.go]: why is it `uint32` not `uint64`?

Comment (taiyangc) [reply splitting.go]: seems that we are truncating existing `uint64` numbers into `uint32`

File: splitting.go [hunk 2]
```diff
@@ -21,6 +21,7 @@ const DefaultBlockSize int64 = 1024 * 256
 type Splitter interface {
 	Reader() io.Reader
 	NextBytes() ([]byte, error)
+	Chunksize() uint64
```
Comment (taiyangc) [inline splitting.go]: Sorry, can we make this `ChunkSize()` ? matches `NextBytes()` naming

Comment (steveyeom) [reply splitting.go]: k.

**Review by taiyangc — APPROVED** [review]

**Pull request merged**